tests/lldp/test_lldp.py

```diff
@@ -238,14 +238,16 @@ def test_lldp_neighbor_post_swss_reboot(duthosts, enum_rand_one_per_hwsku_fronte
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
 
     if loganalyzer:
-        # Ignore all ERR messages, as it is expected many error messages will be generated during swss restart
-        loganalyzer[enum_rand_one_per_hwsku_frontend_hostname].ignore_regex.extend([
-            ".*ERR.*",  # Ignore all ERROR messages
-        ])
-        # Test should fail if LLDP 'unable to send packet on' errors are found
-        loganalyzer[enum_rand_one_per_hwsku_frontend_hostname].match_regex.extend([
-            ".*WARNING lldp#lldpd.*unable to send packet on real device for.*No such device or address"
-        ])
+        # Apply ignore pattern to ALL devices in the testbed
+        for dut in duthosts:
+            # Ignore all ERR messages, as it is expected many error messages will be generated during swss restart
+            loganalyzer[dut.hostname].ignore_regex.extend([
+                ".*ERR.*",  # Ignore all ERROR messages
```

Comment:
how about other patterns in `tests/common/plugins/loganalyzer/loganalyzer_common_match.txt`?


```
r, "\.ERR", "crash"
r, "kernel:.*Oops", "kernel:.*hung", "kernel.*oom\s"
r, "kernel:.*scheduling", "kernel:.*panic"
r, "kernel:.*\serr", "kernel:.*allocation", "kernel:.*kill"
r, "kernel:.*kmemleak.*", "kernel:.* Err:"
s, " ERR "

```